services/common/boltzmann/client/index.js

```diff
@@ -0,0 +1,207 @@
+'use strict'
+
+const fetch = require('node-fetch')
+const os = require('os')
+
+const hostname = os.hostname()
+
+const e = encodeURIComponent
+
+module.exports = class Client {
+  constructor ({
+    url = process.env.STORAGE_API_URL,
+    requestId,
+    userAgent,
+    hostname = hostname,
+    logger
+  }) {
+    this.url = url
+    this.requestId = requestId
+    this.userAgent = `${hostname}(${userAgent})`
```

Comment:
Wrap up the incoming requestId and userAgent (so we know which host is sending which user-agent where, in logs.)

Should I add `x-forwarded-for` here as well, so remote address logging works correctly in storage-api?